CRM/Contact/Page/View/UserDashBoard.php

```diff
@@ -214,7 +214,7 @@ public function run() {
    * @return array
    *   (reference) of action links
    */
-  public function &links() {
+  static function &links() {
```

Comment:
Should we do "public static function &links() {" ? The "public/private/protected" and "static" are independent of each other.

 * "public/private/protected" specifies visibility. PHP won't generate warnings if these are omitted, but phpcs will.
 * "static" indicates that the function will be called using class-notation. Old versions of PHP don't complain about this, but newer versions may complain with "Non-static method foo::bar() should not be called statically" if CRM_Contact_Page_View_UserDashBoard::links() calls a non-static function.